skycoin/skywire
# Update transport file logging

Did you run `make format && make check`?

Fixes #1363	

 Changes:	
- Update dmsgfileserver to use API
- Changed transport file logging to save to csv
- Added  folder `./local/transport_logs`
- Dmsghttp server explicitly serves `/privacy.json` and `system.json`
- Dmsghttp now servers everything inside `./local/transport_logs` and `./local/custom`

How to test this PR:
1. Start a vpn-server
2. Connect to it from your visor
3. Create `privacy.json` file
4. From dmsg build and run `./bin/dmsgget dmsg://<pk>:80/system.json` and check output 
5. From dmsg build and run `./bin/dmsgget dmsg://<pk>:80/privacy.json` and check output 
6. From dmsg build and run `./bin/dmsgget dmsg://<pk>:80/2022-10-10.csv`(todays date) and check output
7. Add `hello.txt` to `./local/custom`
8. From dmsg build and run `./bin/dmsgget dmsg://<pk>:80/hello.txt` and check output
